tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -2642,19 +2642,6 @@ def test_is_metrics_extracted_data_is_included(self):
             dataset="metricsEnhanced",
         )
 
-    def test_is_metrics_extracted_data_is_excluded(self):
```

Comment:
This test does nothing useful. I will need to write a new one on a different PR.